src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.StringBuilder.liquid

```diff
@@ -0,0 +1,3 @@
+private static global::System.Text.StringBuilder GetStringBuilder() => new global::System.Text.StringBuilder(8000);
```

Comment:
How common is this scenario? I think if you find a GC allocated StringBuilder to effect your performance, you might not want to use a generated REST client in the first place...
If this is approved though, I think the avarage url length is a lot shorter than 8000 characters.

Reply:
> By the way, how are you supposed to override the implementation of GetStringBuilder() from this class? And `private partial static void ReturnStringBuilder` is an illegal method declaration, you need to make it `static partial void ReturnStringBuilder`.

Replacing the template for generation.